samples/csharp_dotnetcore/46.teams-auth/TeamsAuth.csproj

```diff
@@ -7,7 +7,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
```

Comment:
Are we sure removing this package the bot keeps working as expected, the login, logout, everything?, don't know but maybe this package someone forgot to remove it?

Reply:
The sample was built successfully. Anyway, I'll test all paths locally.